[2022-08-29T13:56:18.213-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: first_dag_v2.first_task manual__2022-08-29T17:56:15.121655+00:00 [queued]>
[2022-08-29T13:56:18.231-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: first_dag_v2.first_task manual__2022-08-29T17:56:15.121655+00:00 [queued]>
[2022-08-29T13:56:18.232-0400] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T13:56:18.232-0400] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2022-08-29T13:56:18.232-0400] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T13:56:18.270-0400] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): first_task> on 2022-08-29 17:56:15.121655+00:00
[2022-08-29T13:56:18.275-0400] {standard_task_runner.py:52} INFO - Started process 2921 to run task
[2022-08-29T13:56:18.282-0400] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag_v2', 'first_task', 'manual__2022-08-29T17:56:15.121655+00:00', '--job-id', '12', '--raw', '--subdir', '/home/augenz/PycharmProjects/airflow/dags/first_dag_v2.py', '--cfg-path', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmpfjxci73l', '--error-file', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmphfufab_0']
[2022-08-29T13:56:18.284-0400] {standard_task_runner.py:80} INFO - Job 12: Subtask first_task
[2022-08-29T13:56:18.441-0400] {task_command.py:371} INFO - Running <TaskInstance: first_dag_v2.first_task manual__2022-08-29T17:56:15.121655+00:00 [running]> on host pop-os.localdomain
[2022-08-29T13:56:18.662-0400] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=first_dag_v2
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-08-29T17:56:15.121655+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-29T17:56:15.121655+00:00
[2022-08-29T13:56:18.663-0400] {subprocess.py:62} INFO - Tmp dir root location: 
 /run/user/1000/app/com.jetbrains.PyCharm-Community
[2022-08-29T13:56:18.664-0400] {subprocess.py:74} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2022-08-29T13:56:18.677-0400] {subprocess.py:85} INFO - Output:
[2022-08-29T13:56:18.679-0400] {subprocess.py:92} INFO - hello world, this is the first task!
[2022-08-29T13:56:18.680-0400] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-29T13:56:18.807-0400] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=first_dag_v2, task_id=first_task, execution_date=20220829T175615, start_date=20220829T175618, end_date=20220829T175618
[2022-08-29T13:56:18.901-0400] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-29T13:56:19.020-0400] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
